src/dns/coredns.rs

```diff
@@ -227,6 +227,14 @@ impl CoreDns {
                                 reply(sender, src_address, &req);
                             } else {
                                 debug!("Not found, forwarding dns request for {:?}", name);
+                                match env::var("AARDVARK_NO_PROXY") {
+                                    Ok(_) => {
+                                        let mut nx_message = req.clone();
+                                        nx_message.set_response_code(ResponseCode::NXDomain);
+                                        reply(sender.clone(), src_address, &nx_message).unwrap();
```

Comment:
Should there be a return here or something, otherwise we still call `tokio::spawn(async move { ...`  below

Reply:
fixed